client/signin.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import Signup from './signup.jsx';
+import ReactDOM from 'react-dom';
+//import'whatwg-fetch';
+
+function postSend(url, data) {
+  return fetch(url, {
+    credentials: 'same-origin',
+    mode: 'cors', 
```

Comment:
line padding is inconsistent throughout. a single tab must be 4 space. change the setting in your VSCode
There might be a functionality that does the auto indenting for you. search google